docs/reference/data/understanding-database.md

```diff
@@ -0,0 +1,54 @@
+# Understanding Database
+
+By default, n8n uses SQLite as the database. The credentials, workflows, executions, and webhook are decoupled and store in different tables.
+
+## Credentials Entity
+
+The credentials get stored in the `credentials_entity` table with the following fields.
```

Comment:
Can you add a bit more information here? Credentials of n8n or nodes? Can you also link words like credentials to the appropriate entry in the glossary?